.github/workflows/build.yml

```diff
@@ -8,25 +8,22 @@ on:
     branches: [ master ]
 
 jobs:
-  test:
+  build:
     strategy:
       fail-fast: false
-      matrix:
-        platform: [ ubuntu-latest, macos-latest, windows-latest ]
-        ruby: [ 2.7 ]
-    runs-on: ${{ matrix.platform }}
+    runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
         with:
           lfs: true
           fetch-depth: 0
-      - name: Fetch all tags for versioning
+      - name: Fetch all tags for versioning 
         run: git fetch origin +refs/tags/*:refs/tags/*
-      - name: Setup Ruby ${{ matrix.ruby }}
+      - name: Setup Ruby 2.7
```

Comment:
This should be fine on the matrix right?

Reply:
Worked this into a matrix for the build job. It'll be easier to change the ruby version that way. The ruby version for the test jobs is still separate from the ruby for the build job.